added releases notes and docs update for hybrid agents. (#16446)
newrelic/docs-website: src/content/docs/release-notes/mobile-release-notes/flutter-release-notes/flutter-agent-107.mdx

---
subject: Flutter agent
releaseDate: '2023-12-20'
version: 1.0.5
downloadLink: 'https://pub.dev/packages/newrelic_mobile/versions/1.0.6'
---

> # New in this release
> * Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is enabled by default.
> * Updated the native Android agent to version 7.3.0.
> * Updated the native iOS agent to version 7.4.9.
> * fixes for Go Router Instrusmentation

Reviewer: ```suggestion
## New in this release

* Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is enabled by default.
* Updated the native Android agent to version 7.3.0.
* Updated the native iOS agent to version 7.4.9.
* Included fixes for Go Router Instrumentation
```

---
subject: Flutter agent
releaseDate: '2023-12-20'
version: 1.0.5
downloadLink: 'https://pub.dev/packages/newrelic_mobile/versions/1.0.6'
---

## New in this release

> * Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is enabled by default.

Reviewer: ```suggestion
* Introduced [`setMaxOfflineStorageSize`](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage/#flutter) API: This new API allows the user to determine the maximum volume of data that can be stored locally. This aids in better management and control of local data storage.
* Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is enabled by default.
```

---
subject: Flutter agent

> releaseDate: '2023-12-20'

Reviewer: ```suggestion
releaseDate: '2024-03-06'
```

Reviewer: @ndesai-newrelic Can you confirm this is correct?
